README.md

```diff
@@ -6,3 +6,34 @@
 ### Build a project and run integration tests in testsuite module
   - mvn clean verify -DtestsuiteContainer=/path/to/EAP-6.4/dir
   - You can exclude tests, which use remote services, using -PexcludeRemoteTests
+
+### Deploy/work with local jboss instance
+
+  - make sure your instance of jboss is running (/path/to/EAP-6.4/bin/standalone.sh)
+
+  - DEPLOY: mvn clean package jboss-as:deploy -PdeployApplicationEar
+      - deploys 'application' module (application/target/dependency-analysis.ear)
+      - deploys to standalone/data instead of standalone/deployments, this is normal
+      - default endpoints http://localhost:8080/da/ http://localhost:8080/da-bcg/
+      - swagger provides a generated UI to test the endpoints http://localhost:8080/da/doc/doc http://localhost:8080/da-bcg/doc
+
+  - UNDEPLOY: mvn jboss-as:undeploy -PdeployApplicationEar
+  - NOTE: deploying through IDE via 'Run on Server' (e.g. JBDS) seems to corrupt expected endpoints
+
+### Run/debug integration tests in 'testsuite' through IDE
+
+  - modify testsuite/src/test/resources/arquillian.xml, setting:
+      - ${jboss.port} to 9154
+      - ${jboss.home} to /path/to/EAP-6.4/dir
+      - keep these changes for yourself/do not push
```

Comment:
How would you suggest running a single test method from an IDE without modifying arquillian.xml? I understand the variable substitution is done under a maven build, what if we want to execute as a single JUnit test within an IDE?